File: Question1-Two_Sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        array = []
        for a in range(len(nums)-1):
            for b in range(a+1,len(nums)):
                if (nums[a]+nums[b] == target):
                    array.append(a)
                    array.append(b)
                    return array

File: Question1-Two_Sum/two_sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    int *array = malloc(2*sizeof(int));
    *returnSize = 2;
    for(int a = 0;a<numsSize-1;a++ ){
        for(int b = a+1;b<numsSize;b++){
            if ((nums[a]+nums[b]) == target){
                array[0]= a;
                array[1]= b;
                return array;
            }
        }
    }
    return 0;
}

